README.md

```diff
@@ -303,3 +303,13 @@ See the Github project [issues](https://github.com/envato/double_entry/issues).
     ```sh
     bundle exec rake
     ```
+
+## Contributors
+
+Many thanks to those who have contributed to both this gem, and the library upon which it was based, over the years:
+  * Ryan Allen (ryan@ryanface.com)
+  * Keith Pitt (me@keithpitt.com)
+  * Martin Spickermann (spickemann@gmail.com)
+  * Stephanie Staub (staub.steph@gmail.com)
+  * Eaden McKee (eaden@eaden.net)
```

Comment:
not sure if adding the emails is a good idea. maybe link to the githoob profile instead?